packages/venia-ui/lib/components/Checkout/addressForm.js

```diff
@@ -140,18 +128,10 @@ const AddressForm = props => {
                         />
                     </Field>
                 </div>
-                <div className={classes.validation}>{validationMessage}</div>
             </div>
             <div className={classes.footer}>
-                <Button className={classes.button} onClick={cancel}>
-                    Cancel
-                </Button>
-                <Button
-                    className={classes.button}
-                    type="submit"
-                    priority="high"
-                    disabled={submitting}
-                >
+                <Button onClick={cancel}>Cancel</Button>
+                <Button type="submit" priority="high" disabled={isSubmitting}>
```

Comment:
Lost the `className={classes.button}` on these two `Button`s, not sure if that was intentional

Reply:
Nevermind, I see it removed from the CSS too 👍